contribute/contributing_for_everyone.md

```diff
@@ -67,20 +67,20 @@ If by this point you are convinced that contributing is for you, but don't know
 
 ### Find a README file to edit
 
-There are plenty of documentation files in the Julia-verse on GitHub, and typos are so easy to make! You can help clean up documentation by correcting typos or errors which you find. 
+There are plenty of documentation files in the Julia-verse on GitHub, and typos are so easy to make! You can help clean up documentation by correcting typos or errors which you find.
```

Comment:
We can add as well as update broken links or mentions to packages that are outdated.

Reply:
thanks, have updated